### PR TITLE
HTML/CSS create form project

### PR DESCRIPTION
**Pull request by codyloyd**
closes https://github.com/TheOdinProject/top-meta/issues/42 with a new project

### CODE REVIEW

File: html_css/v2/main-course/project-sign-up-form/project-sign-up-form.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+#### Step 1: Set up and planning
+
+1. Set up your HTML and CSS files with some simple dummy content, just to make sure you have everything linked correctly.
+2. Set up your git repository (refer to past projects if you need a refresher).
+3. Download a full-resolution copy of [the design file](INSERT LINK HERE), and get a general idea for how you're going to need to lay things out in your HTML document.
+
+#### Step 2: Gather Assets
+
+1. The design has a large background-image, so go find and download an image you want to use for that section. The one in the design can be found [on unslpash.com](https://unsplash.com/photos/25xggax4bSA), but feel free to select your own. Be sure to credit the creator of your image!
```
Comment (kashura) [inline html_css/v2/main-course/project-sign-up-form/project-sign-up-form.md:18]: Need Oxford coma after "find" (personal preference)

**Review by kashura — COMMENTED** [review]
My only suggestion.

File: html_css/v2/main-course/project-sign-up-form/project-sign-up-form.md (hunk 1 above)
Comment (codyloyd) [reply html_css/v2/main-course/project-sign-up-form/project-sign-up-form.md:18]: 🤔 isn't the oxford comma for lists?

Comment (kashura) [reply html_css/v2/main-course/project-sign-up-form/project-sign-up-form.md:18]: i'll excuse myself, no need to escort me

Comment (twalton83) [inline html_css/v2/main-course/project-sign-up-form/project-sign-up-form.md:18]:
```suggestion
1. The design has a large background-image, so go find and download an image you want to use for that section. The one in the design can be found [on unsplash.com](https://unsplash.com/photos/25xggax4bSA), but feel free to select your own. Be sure to credit the creator of your image!
```

**Pull request merged**